<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pi5009TestWidget</class>
 <widget class="QWidget" name="Pi5009TestWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Select View</string>
   </property>
   <widget class="QPushButton" name="frontCamButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Front Cam</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rearCamButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Rear Cam</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leftCamButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Left Cam</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightCamButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Right Cam</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quadViewButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Quad View</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>931</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutPcbConsole"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
